model/schema/phenotypeAndDiseaseAnnotation.yaml

```diff
@@ -352,9 +352,7 @@ classes:
       condition_taxon:
         description: >-
           NCBITaxon ontology id used in subset of condition types like 'bacterial infection'.
-        range: OntologyTerm
-        values_from:
-          - NCBITaxon
+        range: NCBITaxonTerm
```

Comment:
actually don't need this range constraint as it is redundant with the condition_taxon slot definition constraint (probably my own oversight when first writing the model)